s32/mcux/devices/S32Z270/S32Z270_glue_mcux.h

```diff
@@ -225,5 +225,25 @@
                                                     RTU_CE_CAN20_MB_127_96_IRQn, RTU_CE_CAN21_MB_127_96_IRQn, RTU_CE_CAN22_MB_127_96_IRQn, RTU_CE_CAN23_MB_127_96_IRQn }
 
 #define CAN_ORed_Message_buffer_IRQS                CAN_ORed_Message_buffer_0_31_IRQS
+/*!
```

Comment:
```suggestion

/*!
```
a blank line to separate from previous block :)